scripts/__input_initialize/__input_initialize.gml

```diff
@@ -170,13 +170,36 @@ function __input_initialize()
         ++_p;
     }
     
+    //DualSense haptic trigger effect types
+    enum INPUT_TRIGGER_EFFECT
```

Comment:
Is this enum expected to be public? If not then please prefix with __ (same with the enum members as well i.e. `__INPUT_TRIGGER_EFFECT.__TYPE_OFF`)